src/Arcus.WebApi.Unit/Arcus.WebApi.Unit.csproj

```diff
@@ -1,34 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Would it be possible to sue the same indenting or not?
This is nitpicking but not fully sure what changed here.

Reply:
Yeah, sorry; was formatted in vscode to 2-space indentation instead of 4-space. I'll revert this.

Reply:
No problem!